paddlex/inference/models/text_to_speech_acoustic/processors.py

```diff
@@ -0,0 +1,173 @@
+# copyright (c) 2024 PaddlePaddle Authors. All Rights Reserve.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# import lazy_paddle as paddle
+from typing import Dict
+from typing import List
+from typing import Optional
+from typing import Sequence
+from typing import Tuple
+
+import os
+import paddle
+import numpy as np
+from paddle import inference
+from paddle import nn
+import paddlespeech
+import paddlespeech.utils
+
+def get_predictor(
+        model_dir: Optional[os.PathLike]=None,
+        model_file: Optional[os.PathLike]=None,
+        params_file: Optional[os.PathLike]=None,
+        device: str='cpu',
+        # for gpu
+        use_trt: bool=False,
+        device_id: int=0,
+        # for trt
+        use_dynamic_shape: bool=True,
+        min_subgraph_size: int=5,
+        # for cpu
+        cpu_threads: int=1,
+        use_mkldnn: bool=False,
+        # for trt or mkldnn
+        precision: int="fp32"):
+    """
+    Args:
+        model_dir (os.PathLike): root path of model.pdmodel and model.pdiparams.
+        model_file (os.PathLike): name of model_file.
+        params_file (os.PathLike): name of params_file.
+        device (str): Choose the device you want to run, it can be: cpu/gpu, default is cpu.
+        use_trt (bool): whether to use TensorRT or not in GPU.
+        device_id (int): Choose your device id, only valid when the device is gpu, default 0.
+        use_dynamic_shape (bool): use dynamic shape or not in TensorRT.
+        use_mkldnn (bool): whether to use MKLDNN or not in CPU.
+        cpu_threads (int): num of thread when use CPU.
+        precision (str): mode of running (fp32/fp16/bf16/int8).  
+    """
+    rerun_flag = False
+    if device != "gpu" and use_trt:
+        raise ValueError(
+            "Predict by TensorRT mode: {}, expect device=='gpu', but device == {}".
+            format(precision, device))
+    
+    # after paddle 3.0, support new inference interface
+    if paddlespeech.utils.satisfy_paddle_version('3.0.0-beta'):
```

Comment:
paddlex infer will not install paddlespeech. Also paddlex will install paddle>=3.0, therefore version check is unnecessary.